hw4.Rmd

```diff
@@ -30,43 +35,98 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+dat.decide.all <- dat %>%
+  group_by(judge) %>%
```

Comment:
For this question, we are just interested in the proportion for all of the judges overall so group_by is not necessary. See solutions.